### PR TITLE
[WIP] GUI-less training example script

### PR DESCRIPTION
**Pull request by C-Achard**
Adds :

- A Dockerfile draft for CellSeg3D
- A script for training without napari

### CODE REVIEW

**Review by CYHSM — APPROVED** [review]
LGTM!

**Comment by C-Achard** [comment]
> LGTM!

Thanks a lot @CYHSM ! I will merge once the Dockerfile is approved as well.

**Comment by C-Achard** [comment]
No news on Dockerfile so it will be omitted for now

**Pull request merged**
